keras_cv/models/backbones/efficientnet_v2/efficientnet_v2_backbone.py

```diff
@@ -0,0 +1,551 @@
+        x = layers.BatchNormalization(
+            axis=BN_AXIS,
+            momentum=bn_momentum,
+            name="stem_bn",
+        )(x)
+        x = layers.Activation(activation, name="stem_activation")(x)
+
+        # Build blocks
+        blocks_args = copy.deepcopy(blocks_args)
+        b = 0
+        blocks = float(sum(args["num_repeat"] for args in blocks_args))
+
+        for i, args in enumerate(blocks_args):
+            assert args["num_repeat"] > 0
+
+            # Update block input and output filters based on depth multiplier.
+            args["input_filters"] = round_filters(
+                filters=args["input_filters"],
+                width_coefficient=width_coefficient,
+                min_depth=min_depth,
+                depth_divisor=depth_divisor,
+            )
+            args["output_filters"] = round_filters(
+                filters=args["output_filters"],
+                width_coefficient=width_coefficient,
+                min_depth=min_depth,
+                depth_divisor=depth_divisor,
+            )
+
+            repeats = round_repeats(
+                repeats=args.pop("num_repeat"),
+                depth_coefficient=depth_coefficient,
+            )
+            for j in range(repeats):
+                # The first block needs to take care of stride and filter size
+                # increase.
+                if j > 0:
+                    args["strides"] = 1
+                    args["input_filters"] = args["output_filters"]
+
+                # Determine which conv type to use:
+                block = {
+                    0: MBConvBlock(
+                        input_filters=args["input_filters"],
+                        output_filters=args["output_filters"],
+                        expand_ratio=args["expand_ratio"],
+                        kernel_size=args["kernel_size"],
+                        strides=args["strides"],
+                        se_ratio=args["se_ratio"],
+                        activation=activation,
+                        bn_momentum=bn_momentum,
+                        survival_probability=drop_connect_rate * b / blocks,
+                        name="block{}{}_".format(i + 1, chr(j + 97)),
+                    ),
+                    1: FusedMBConvBlock(
+                        input_filters=args["input_filters"],
+                        output_filters=args["output_filters"],
+                        expand_ratio=args["expand_ratio"],
+                        kernel_size=args["kernel_size"],
+                        strides=args["strides"],
+                        se_ratio=args["se_ratio"],
+                        activation=activation,
+                        bn_momentum=bn_momentum,
+                        survival_probability=drop_connect_rate * b / blocks,
+                        name="block{}{}_".format(i + 1, chr(j + 97)),
+                    ),
+                }[args["conv_type"]]
+
+                x = block(x)
+                b += 1
+
```

Comment:
`pyramid_level_inputs[i+2] = x.node.layer.name`

Reply:
doesn't work because some blocks don't downsample so the i index doesn't match up with pyramid level idnices